cargo-dist/src/sign/macos.rs

```diff
@@ -0,0 +1,178 @@
+use std::path::PathBuf;
```

Comment:
It'd be nice to have a top-level module comment here of like the general flow/idea, or any Exciting Issues this code deals with.